public/scss/navigation.scss

```diff
@@ -12,6 +12,7 @@
   top: 0;
   background: #fff;
   border-bottom: 1px solid #00000017;
+  box-shadow: 0 0 25px #dcdcdc;
```

Comment:
Doesn't work for dark mode. 

![image](https://user-images.githubusercontent.com/6109680/66823915-c4170600-ef0c-11e9-819d-f03925fc9b58.png)

As a general rule, we've always avoided box shadows. I think it kinda works here though. Thoughts @chappjc? My thinking is partially based on guidance from @gozart1 from https://github.com/decred/dcrdata/pull/921#issuecomment-455096066

Reply:
I certainly don't want my browser eating more CPU than needed, but I've not qualified the assertion that it is more demanding to render.

Regardless, the shadow is OK, although about twice as big as it needs to be.